docs/conf.py

```diff
@@ -26,6 +26,7 @@
 from sphinx.application import Sphinx
 
 sys.path.insert(0, os.path.abspath(".."))
+sys.path.insert(0, os.path.abspath("."))
```

Comment:
Like it or not, this is probably the simplest way of making a package living in `docs` be importable :(

Reply:
Yeah, this is weird.  I'm curious if it's related to the directory change done by `tox`.  Anyways, we will eventually break `automodapi` out into it's own package, so I'm not going to fret about it for now.